Tests/IntegrationTests/PropagationTestHelper.hpp

```diff
@@ -309,7 +309,7 @@ std::pair<Vector3D, double> to_cylinder(
 }
 
 // test propagation to most surfaces
-template <typename Propagator_type, typename Surface_type>
+template <typename Propagator_type, typename SurFaceType>
```

Comment:
I'd say we write 

```suggestion
template <typename Propagator_type, typename SurfaceType>
```

rather than `SurFaceType`. That's probably a typo? This goes for the whole file I guess.

Reply:
That's a typo indeed - or actually a global replacement error. I changed `face_type` to `FaceType` ... Doh.